app/models/ebean/ReportExecution.java

```diff
@@ -37,7 +38,7 @@
 
 /**
  * An execution event for a {@link Report}.
- *
+ * <p>
```

Comment:
nit: missing `</p>`?

Reply:
The javadoc compiler automatically inserts closing paragraph tags